forcats.Rmd

```diff
@@ -228,24 +228,24 @@ countries_in_1896 %>%
 ```
 
 ### Reordering by appearance 
-The package forcats offers with the function `fct_inorder()` the possibility to orientate the order of the levels of a factor to the occurence of the corresponding values in the data set. To make this clear, we take a look at the first 15 entries in the corresponding data set:
+The package forcats contains the function `fct_inorder()` which enables the sorting of levels by their order of appearance in the data set. To make this clear, we look at the first 20 entries of the data set:
 
 ```{r}
 olympic_1896$country %>%
-  head(15)
+  head(20)
 ```
 
 Create a factor with levels in the order of the appearance shown above:
 
 ```{r}
-olympic_1896$country %>%
+olympic_1896$country[1:20] %>%
```

Comment:
This will not lead to the desired output. 
We only used the restriction of 20 entries to keep the amount of what's printed reasonable in the code chunk before.
Here we want to show all levels of the `olympic_1896$country` variable.
But I agree that this point is arguable.

Anyway, here is what happens with your change and in the original:
```R
# Your approach
olympic_1896$country[1:20] %>%
    fct_inorder() %>%
    levels()
[1] "Greece"        "Great Britain" "Switzerland"   "United States" "Germany" 

# Intended:
olympic_1896$country %>%
    fct_inorder() %>%
    levels()
 [1] "Greece"        "Great Britain" "Switzerland"   "United States" "Germany"       "France"       
 [7] "Hungary"       "Australia"     "Austria"       "Denmark"       "Italy"         "Sweden" 
```

Reply:
At this point, I feel the same way Carl does. Could you change that point one more time, Bente?